plugin/closetag.vim

```diff
@@ -21,18 +23,33 @@ fun! s:Initial()
     if g:closetag_filenames != ''
         if g:closetag_shortcut != ''
             exec "au BufNewFile,Bufread " . g:closetag_filenames . " inoremap <silent> <buffer> " . g:closetag_shortcut . " ><Esc>:call <SID>CloseTagFun()<Cr>"
-            exec "au User vim-closetag inoremap <silent> <buffer> " . g:closetag_shortcut . " ><Esc>:call <SID>CloseTagFun()<Cr>"
+            exec "au! User vim-closetag inoremap <silent> <buffer> " . g:closetag_shortcut . " ><Esc>:call <SID>CloseTagFun()<Cr>"
```

Comment:
Do not use `au!` here for deleting autocommand.
It's for some other plugins lazy loading, you can check this closed issue:
https://github.com/alvan/vim-closetag/issues/9